.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-         python-version: ['3.8', '3.12']
+         python-version: ['3.9', '3.12']
 
     steps:
     - uses: actions/checkout@v3
```

Comment:
@MoisesGSalas should we bump this to [v4](https://github.com/actions/checkout/releases/tag/v4)?